Sources/Keychain/commons/Platform.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+struct Platform {
+
+    static var isRealDevice: Bool {
+        return TARGET_OS_SIMULATOR == 0
```

Comment:
You likely want `#if targetEnvironment(simulator)` instead

Reply:
makes total sense, changed